src/main/java/org/spongepowered/common/mixin/api/mcp/entity/AreaEffectCloudEntityMixin_API.java

```diff
@@ -83,7 +84,7 @@ public AreaEffectCloudData getAreaEffectCloudData() {
     }
 
     @Override
-    public Mutable<Color> color() {
+    public Value.Mutable<Color> color() {
```

Comment:
Remove these override that implement data methods. All data methods in the API are to be default implemented so that it all flows through the DataProvider system. If you find any interfaces that return a data instance that aren't default implemented, do us up a PR to fix that.